shared/default-data/index.ts

```diff
@@ -51,3 +56,47 @@ export function getDefaultData(): ElementConfigType {
     widgetTriggerIcon: ''
   }
 }
+
+export const createDefaultBannerConfig = (
+  presetName: string
+): BannerConfig => ({
+  $version: '1.0.0',
+  $name: presetName,
+  $modifiedAt: new Date().toISOString(),
```

Comment:
Let's not add this in defaults. That we can, we can use it to compare if the server version is different from local version - if no `$modifiedAt` is set, it's a fresh local config.